decidim-assemblies/app/presenters/decidim/assemblies/assembly_stats_presenter.rb

```diff
@@ -13,22 +13,12 @@ def collection
         highlighted_stats.concat(assembly_followers_stats(priority: StatsRegistry::HIGH_PRIORITY))
         highlighted_stats.concat(component_stats(priority: StatsRegistry::HIGH_PRIORITY))
         highlighted_stats.concat(component_stats(priority: StatsRegistry::MEDIUM_PRIORITY))
+        highlighted_stats.concat(comments_stats(:assemblies, tag: :comments))
```

Comment:
💬 I have a doubt: why are you adding the `tag: :comments` condition if you are only retrieving the `:comments_count` stat? It isn't redundant?